modules/EnsEMBL/Web/Component/Location/Compara_AlignSliceBottom.pm

```diff
@@ -91,7 +92,7 @@ sub content {
       more_slices     => $i != @$slices,
     });
     
-    my ($species_name, $slice_name) = split ':', $species_defs->production_name_mapping($_->{'name'});
+    my ($species_name, $slice_name) = split ':', $_->{'name'};
```

Comment:
This looks like it should be like this:
`$lookup->{$_->{'name'}}`

Reply:
That's what I assumed when I did my big update - but when I looked at it on the website, the label was "Ancestral sequence" because it didn't recognise it from the URL. Everything is interconnected with this code, which is why a PR review alone can't guarantee it's correct.